concrete/src/Html/Object/Picture.php

```diff
@@ -31,39 +31,73 @@ class Picture extends Element
     //////////////////////////// CORE METHODS //////////////////////////
     ////////////////////////////////////////////////////////////////////
 
-    public function __construct(array $sources = array(), $fallbackSrc, $attributes = array())
+    public function __construct(array $sources = array(), $fallbackSrc, $attributes = array(), $lazyLoadNative, $lazyLoadJavaScript)
     {
-        $this->sources($sources);
-        $this->fallback($fallbackSrc);
+        $this->sources($sources, $lazyLoadJavaScript);
+
+        if ($lazyLoadJavaScript) {
+            $this->noScriptFallback($fallbackSrc, $lazyLoadNative);
+        }
+
+        $this->fallback($fallbackSrc, $lazyLoadNative, $lazyLoadJavaScript);
     }
 
     /**
      * Static alias for constructor.
      *
-     * @param string $element
-     * @param string|null|Tag $value
+     * @param array $sources
+     * @param string|null $fallbackSrc
      * @param array $attributes
+     * @param bool|null $lazyLoadNative
+     * @param bool|null $lazyLoadJavaScript
      *
-     * @return                Table
+     * @return \Concrete\Core\Html\Object\Picture
      */
-    public static function create($sources = array(), $fallbackSrc = false, $attributes = array())
+    public static function create($sources = array(), $fallbackSrc = false, $attributes = array(), $lazyLoadNative = false, $lazyLoadJavaScript = false)
     {
-        return new static($sources, $fallbackSrc, $attributes);
+        return new static($sources, $fallbackSrc, $attributes, $lazyLoadNative, $lazyLoadJavaScript);
     }
 
     ////////////////////////////////////////////////////////////////////
     /////////////////////////////// CHILDREN ///////////////////////////
     ////////////////////////////////////////////////////////////////////
 
-    public function sources($sources)
+    /**
+     * Create the source elements for the picture element.
+     *
+     * Example output:
+     * <!--[if IE 9]><video style='display: none;'><![endif]-->
+     * <source srcset="..." media="(min-width: ...)">
+     * <source srcset="..." media="(min-width: ...)">
+     * <source srcset="...">
+     * <!--[if IE 9]></video><![endif]-->
+     * or
+     * <!--[if IE 9]><video style='display: none;'><![endif]-->
+     * <source data-srcset="..." media="(min-width: ...)">
+     * <source data-srcset="..." media="(min-width: ...)">
+     * <source data-srcset="...">
+     * <!--[if IE 9]></video><![endif]-->
+     *
+     * @param array $sources An array of image thumbnail file paths.
+     * @param bool $lazyLoadJavaScript If true, the source image thumbnail file path is set to a "data-srcset" attribute instead of "srcset".
+     *
+     * @return \Concrete\Core\Html\Object\Picture
+     */
+    public function sources($sources, $lazyLoadJavaScript)
```

Comment:
The new parameter should have a default, so preserving compatibility with other uses that may not have implemented that parameter